client/views/edit-event-assignment.js

```diff
@@ -62,44 +53,66 @@ module.exports = View.extend({
   getOptions: function () {
     var species = this.model.collection.parent.collection.parent.species;
     var parameters = this.model.collection.parent.collection.parent.parameters;
-    var speciesNames = species.map(function (specie) { return specie.name });
-    var parameterNames = parameters.map(function (parameter) { return parameter.name });
-    return speciesNames.concat(parameterNames);
+    var specs = species.map(function (specie) {
```

Comment:
Consider removing the "Variable" or "Parameter" options groups when there are no available variables/parameters